### ISSUE TITLE
Apigee envgroup update not working

### ISSUE THREAD

**Issue by cosmin-constantin-oprea-db** [hashicorp/terraform-provider-google#9696]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave _+1_ or _me too_ comments, they generate extra noise for issue followers and do not help prioritize the request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If an issue is assigned to the `modular-magician` user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to `hashibot`, a community member has claimed the issue already.

### Terraform Version

0.13.3

### Affected Resource(s)

google_apigee_envgroup

* google_XXXXX

### Terraform Configuration Files

```tf
resource "google_apigee_envgroup" "id" {
  org_id    = google_apigee_organization.org.id
  name      = var.env_group_name
  hostnames = var.hostnames
}
```

### Debug Output

Looks like the request made in the resource code is a PUT:
https://github.com/hashicorp/terraform-provider-google/blob/c1b95fc11ea06c045903c8fdb4f4bef25673554c/google/resource_apigee_envgroup.go#L214
The expected request should be PATCH:
https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.envgroups/patch

### Expected Behavior

Env group should update with the new hostnames.

### Actual Behavior

Getting a 404 response in the logs.

### Steps to Reproduce

Create an envgroup and in a new run try to update the hostnames.

**Comment by ScottSuarez** [comment]
confirming with `apigee` team who added support for this feature

**Comment by odise** [comment]
@ScottSuarez could you please indicate when this bugfix will be released? It isn't part of 3.79.0 as far as I can see.

**Comment by ScottSuarez** [comment]
It will release on Monday with the next release pending no issues
